engines/router/missionctl/log/resultlog/kafka_test.go

```diff
@@ -165,27 +163,14 @@ func TestKafkaLoggerWrite(t *testing.T) {
 		topic:               "test-topic",
 		producer:            mp,
 	}
-	testKafkaLogEntry := []byte(`{"key": "value"}`)
-	turingResLogEntry := &TuringResultLogEntry{
-		resultLogMessage: turing.TuringResultLogMessage{},
-	}
-
-	// Patch newKafkaLogEntry
-	monkey.Patch(
```

Comment:
no reason to patch `newJSONKafkaLogEntry`, can just use its existing implementation and the assertion below will validate it